apps/docs/src/pages/en/sdks/go.mdx

```diff
@@ -42,39 +42,42 @@ package main
 import (
 	"context"
 	"fmt"
-	"time"
 
 	"github.com/streamdal/go-sdk"
 )
 
 func main() {
-	ctx, cancel := context.WithCancel(context.Background())
-	defer cancel()
-
-	sc, err := streamdal.New(&streamdal.Config{
-		ServerURL:       "localhost:8082",
-		ServerToken:     "streamdal",
-		StepTimeout:     time.Millisecond * 10,
-		PipelineTimeout: time.Millisecond * 100,
-		DryRun:          false,
+	sc, _ := streamdal.New(&streamdal.Config{
+		// Address of the streamdal server
+		ServerURL:       "streamdal-server.svc.cluster.local:8082",
+
+		// Token used for authenticating with the streamdal server
+		ServerToken:     "1234",
+
+		// Identify _this_ application/service (
 		ServiceName:     "billing-svc",
-		ShutdownCtx:     ctx,
```

Comment:
Is `ShutdownCtx` not needed anymore? I thought it error'd if it wasn't provided.